exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -179,12 +207,47 @@ public String getMetricsTable() {
       }
       for (final Number value : values) {
         if (value != null) {
-          builder.appendFormattedNumber(value, null);
+          builder.appendFormattedNumber(value);
         } else {
-          builder.appendCell("", null);
+          builder.appendCell("");
         }
       }
     }
     return builder.build();
   }
+
+  private class OverviewTblTxt {
+    static final String OperatorID = "Operator ID";
```

Comment:
I'll change it to the Java convention. It was getting to be jarring to see everything in Upper case when defining the columns. Should follow the convention. Thanks!